addons/auth_passkeys/models/users_passkey.py

```diff
@@ -0,0 +1,90 @@
+from odoo import fields, models, api, _
+from odoo.addons.base.models.res_users import check_identity
+from odoo.exceptions import AccessDenied
+
+class UsersPasskey(models.Model):
+    _inherit = "res.users"
+
+    auth_passkeys_key_ids = fields.One2many("auth.passkeys.key", "create_uid")
+    password_login_disabled = fields.Boolean(string="Password Disabled",
+                                            compute="_compute_password_login_disabled",
+                                            readonly=True,
+                                            help="Only allow users to login via Passkeys.")
+    invalidation_counter = fields.Integer(default=0)
+
+    @property
+    def SELF_WRITEABLE_FIELDS(self):
+        return super().SELF_WRITEABLE_FIELDS + ['auth_passkeys_key_ids']
```

Comment:
Why is this necessary ? Shouldn't a user always go through the passkey creation wizard to create a new key and through a revoke method button to disable one ? Instead of writing directly to `auth_passkeys_key_ids`?

Giving the ability to users to directly write on `auth_passkeys_key_ids` opens an attacker vector.